localdev/seed/utils.py

```diff
@@ -0,0 +1,10 @@
+import re
```

Comment:
I think a better choice would be to make a valid regex instead invalidation regex. But at this point i feel that we should only make the changes in the `.JSON` file above and leave/remove the rest for now.

We will(Should) at some point add a validation function for course keys in https://github.com/mitodl/ol-django/tree/main/src/mitol/openedx using edx_opaque keys package so that the validation stays the same between our apps and the open edX. 

Later on, we will just use that in our applications wherever we would need to validate a course key.